packages/dx-react-grid/src/plugins/table-fixed-columns.jsx

```diff
@@ -20,6 +20,14 @@ const CellPlaceholder = props => <TemplatePlaceholder params={props} />;
 
 const pluginDependencies = [
   { name: 'Table' },
+  { name: 'TableBandHeader', optional: true },
+  { name: 'TableColumnReordering', optional: true },
+  { name: 'TableEditColumn', optional: true },
+  { name: 'TableEditRow', optional: true },
+  { name: 'TableFilterRow', optional: true },
+  { name: 'TableHeaderRow', optional: true },
+  { name: 'TableSelection', optional: true },
+  { name: 'TableTreeColumn', optional: true },
```

Comment:
What about the `TableSummaryRow` plugin? Next behavior looks like a bug

![image](https://user-images.githubusercontent.com/15093892/49784475-7ea70c80-fd2e-11e8-99e9-6fabc514ce80.png)